chainweb.cabal

```diff
@@ -539,32 +538,22 @@ executable chainweb-miner
           chainweb
 
         -- external
-        , aeson >= 1.4
-        , async >= 2.2
         , base >= 4.11 && < 5
-        , base-prelude >= 1.3
-        , base16-bytestring >= 0.1
-        , bytes >= 0.15
-        , bytestring >= 0.10
         , connection >= 0.2
-        , containers >= 0.5
         , errors >= 2.3
+        , generic-lens >= 1.2
         , http-client >= 0.5
         , http-client-tls >= 0.3
         , mwc-random >= 0.13
         , optparse-applicative >= 0.14
-        , pretty-simple >= 2.1
         , retry >= 0.7
+        , rio >= 0.1.5
```

Comment:
RIO offered the simplest logging infrastructure that I knew how to get running quickly. Its logging has nice looking, reasonable defaults. It cleans up a lot of code below, too.

Reply:
Yet another logging system?  Not a fan of that.  Also, rio seems like overkill here.  Let's stick with the logging system we've already got.  Either that or print statements are perfectly fine.

Reply:
I definitely thought the decision over. Here were my thoughts:

- Chainweb's logging is somewhat complicated, is tailored for Chainweb, is an amalgamation of 4 different module groups (`yet-another-logger` + `loglevel` + `Chainweb.Logger` + `Utils.Logging`), and Lars wants to change it anyway
- `yet-another-logger` states of itself "This Version is yet a preview", Lars tells me it's bit-rotten, it's also a `ReaderT` pattern, and I foresaw it interacting dubiously with the parallelism in the miner
- `rio` is maintained, it's tailored for CLI-based Haskell apps, it has sane defaults, very low boilerplate, easy to use, plays nice with the rest of the ecosystem, I already knew how to use it, and I had a single sitting to come up with something professional

So I went with it. It's certainly a leap above print statements, especially w.r.t. quality of output and configurable log levels. It was effectively a drop-in replacement, save the refactoring I did of `main` itself.